### PR TITLE
chore(docs): update cognito provider doc

### PR DESCRIPTION
**Pull request by markCwatson**
Requested on [PR#10600](https://github.com/nextauthjs/next-auth/pull/10600) by @ndom91

### CODE REVIEW

**Comment by markCwatson** [comment]
@ndom91 I do not trust my local MDX processor... so not 100% confident on how this looks

**Review by ndom91 — APPROVED** [review]
LGTM to me :wink: 

image

File: docs/pages/getting-started/providers/cognito.mdx [hunk 1]
```diff
@@ -83,6 +84,18 @@ app.use(
   </Code.Express>
 </Code>
 
+### Options
+
+The **Amazon Cognito Provider** comes with a set of default options:
```
Comment (ndom91) [inline docs/pages/getting-started/providers/cognito.mdx:89]: Although this whole part about the options just links to the code for the provider, this is kind of implied.. 

Let's get rid of this part and keep the Callout about the hosted domain settings as well as the link for hosted domain above, thanks!

**Review by ndom91 — CHANGES_REQUESTED** [review]

**Review by ndom91 — APPROVED** [review]

**Comment by markCwatson** [comment]
@ndom91 @ubbe-xyz and I expected to merge latest changes from `main` (and merge?)?

**Comment by ndom91** [comment]
@markCwatson yeah this looks good now, thanks!

**Pull request merged**